pkg/api/packages_test.go

```diff
@@ -127,22 +127,22 @@ func TestUpdatePackage(t *testing.T) {
 }
 
 func TestUpdatePackageFlatcar(t *testing.T) {
+	var err error
 	a := newForTest(t)
 	defer a.Close()
 
 	pkg := &Package{
 		Type:          PkgTypeFlatcar,
 		URL:           "https://commondatastorage.googleapis.com/update-storage.core-os.net/amd64-usr/766.3.0/",
-		Filename:      dat.NullStringFrom("update.gz"),
+		Filename:      null.NewString("update.gz", true),
 		Version:       "2016.6.6",
-		Size:          dat.NullStringFrom("123456"),
-		Hash:          dat.NullStringFrom("sha1:blablablabla"),
+		Size:          null.NewString("123456", true),
+		Hash:          null.NewString("sha1:blablablabla", true),
 		ApplicationID: flatcarAppID,
 	}
-	_, err := a.AddPackage(pkg)
+	pkg, err = a.AddPackage(pkg)
```

Comment:
If you drop the `var err error` above, you can do:

```suggestion
	pkg, err := a.AddPackage(pkg)
```